src/HangoutsChatChannel.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace NotificationChannels;
+
+use GuzzleHttp\Client;
+use Illuminate\Notifications\Notification;
+use Illuminate\Support\Arr;
+use NotificationChannels\Exceptions\CouldNotSendNotification;
+use Psr\Http\Message\ResponseInterface;
+
+class HangoutsChatChannel
+{
+    /** @var Client */
+    protected $client;
+
+    /**
+     * @param Client $client
+     */
+    public function __construct(Client $client)
+    {
+        $this->client = $client;
+    }
+
+    /**
+     * Send the given notification.
+     *
+     * @param mixed $notifiable
+     * @param Notification $notification
+     *
+     * @return ResponseInterface|void
+     *
+     * @throws CouldNotSendNotification
+     */
+    public function send($notifiable, Notification $notification)
+    {
+        if (! $url = $notifiable->routeNotificationFor('hangoutsChat')) {
+            return;
+        }
+
+        $webhookData = $notification->{'toHangoutsChat'}($notifiable)->toArray();
```

Comment:
Worth checking that the method actually exists and that it returns the correct object before calling to array

Reply:
Not sure why is it called this way `$notification->{'toHangoutsChat'}($notifiable)` when you can simply just use `$notification->toHangoutsChat($notifiable)`?